packages/flutter/lib/src/widgets/page_view.dart

```diff
@@ -553,6 +564,13 @@ class PageView extends StatefulWidget {
   /// children because constructing the [List] requires doing work for every
   /// child that could possibly be displayed in the page view, instead of just
   /// those children that are actually visible.
+  ///
+  /// {@template flutter.widgets.pageView.allowImplicitScrolling}
+  /// The [allowImplicitScrolling] parameter must not be null. If true, the
```

Comment:
Also: This setting is ignored if you provide your own physics, right?

Maybe we shouldn't expose this as an extra flag and just say that you can enable this by providing custom physics?

Reply:
(I believe the other scrollable also don't have an extra flag for this)

Reply:
The problem is that we also need to specify a cache extent for it to work on iOS.  Plus we currently swap out your custom physics if you want page snapping.